examples/helloworld/tf_example1/test.py

```diff
@@ -6,6 +6,7 @@
 from neural_compressor.data import DefaultDataLoader
 from neural_compressor.quantization import fit
 from neural_compressor.config import PostTrainingQuantConfig
+from neural_compressor.metric import TensorflowTopK
```

Comment:
why we doesn't support below usage @PenghuiCheng 

from neural_compressor import Metric
eval_metric = Metric('topk', ...)

Reply:
We support method above